frontend/src/modules/otherlocales/components/Translation.js

```diff
@@ -113,7 +114,7 @@ export default function Translation(
                 <p
                     lang={translation.locale.code}
                     dir={translation.locale.direction}
-                    script={translation.locale.script}
+                    data-script={translation.locale.script}
```

Comment:
I've tried to find if this is used anywhere, but I couldn't find it. I did find `data-script` selectors in https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/entitieslist/components/Entity.css#L48-L53, but no corresponding CSS in `otherlocales`. So maybe the way we render them isn't great?

Reply:
Nice catch! There's also https://github.com/mozilla/pontoon/blob/master/frontend/src/index.css#L41, which will now also impact `otherlocales`.